.vscode/launch.json

```diff
@@ -1,11 +1,62 @@
 {
   "version": "0.2.0",
   "configurations": [
-      {
-          "type": "pwa-node",
-          "request": "attach",
-          "name": "attach-to-service",
-          "port": 9229
-      }
+    /**
+     * Attaches a debugger to an already running OSD instance
+     */
+    {
+      "name": "Attach Debugger to running Dashboards Server",
+      "type": "pwa-node",
+      "request": "attach",
+      "port": 9229
+    },
+    /**
+     * Starts the OSD instance only
+     */
+    {
+      "name": "Start Dashboards Server",
+      "type": "node",
+      "request": "launch",
+      "runtimeExecutable": "yarn",
+      "args": ["start", "--no-base-path"],
+      "cwd": "${workspaceFolder}",
+      "runtimeArgs": [],
+      "env": {
+        "NODE_ENV": "development"
+      },
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "openOnSessionStart"
+    },
+    /**
+     * Starts the OpenSearch snapshot server
+     * - Everytime the server is stopped, data will NOT be persisted. Thus, if you need to persist data, you can add a configuration to spin up your OpenSearch cluster of choice
```

Comment:
it's actually flipped.

Everytime the server is started it will overwrite your previous snapshot data.

the files get placed in `.opensearch/{version}`

so if you run `yarn opensearch snapshot`, then stop it and then the next day you can actually re-access that same instance with the data available with by doing `./.opensearch/{version}/bin/opensearch`